lua/entities/gmod_wire_expression2/core/signal.lua

```diff
@@ -163,26 +163,22 @@ __e2setcost(4)
 
 --- Returns the name of the received signal.
 e2function string signalName()
-	if not currentSignal then return "" end
-	return currentSignal[2]
+	return currentSignal and currentSignal[2] or ""
```

Comment:
More replacing fully functional code with `and or` for literally no reason, gotta keep up the kill streak amiright